modules/tinymce/src/plugins/table/main/ts/actions/TableActions.ts

```diff
@@ -63,10 +64,11 @@ export const TableActions = function (editor: Editor, lazyWire) {
         Arr.each(result.newCells(), function (cell) {
           fireNewCell(editor, cell.dom());
         });
-        return result.cursor().map(function (cell) {
+        return result.cursor().map((cell) => {
+          const des = DomDescent.freefallRtl(cell);
```

Comment:
I hope this won't mess with LtR?

Reply:
It's not really related to directionality as I understand it, just which end of the node it should try and dropdown into. So if you had `<p>blah</p>` it'd end up after the `h`.